modules/dcache/src/main/java/org/dcache/services/ssh2/Ssh2Admin.java

```diff
@@ -318,6 +327,66 @@ public boolean authenticate(String userName, PublicKey key,
         }
     }
 
+    private class RemoteHostValidator {
+
+        private boolean isValidHost(String line, ServerSession session) {
```

Comment:
You're absolutely right, I've updated the behavior to combine multiple entries according to this specification. I will also add the unit tests in a seperate PR